src/auto-evo/RunResults.cs

```diff
@@ -368,6 +328,155 @@ public long GetPopulationInPatch(Species species, Patch patch)
             }
         }
 
+        /// <summary>
+        ///   Computes the final population of all species, by patch.
+        /// </summary>
+        public Dictionary<Patch, Dictionary<Species, long>> GetPopulationsByPatch(bool resolveMigrations = false,
+            bool resolveSplits = false, bool includeNewSpecies = true)
+        {
+            var speciesInPatches = new Dictionary<Patch, Dictionary<Species, long>>();
+            foreach (var species in results.Keys)
+            {
+                var populations = GetPopulationsByPatch(species, resolveMigrations, resolveSplits, includeNewSpecies);
+                foreach (var patchEntry in populations)
+                {
+                    if (!speciesInPatches.ContainsKey(patchEntry.Key))
+                        speciesInPatches[patchEntry.Key] = new Dictionary<Species, long>();
+
+                    patchEntry.Value.ToList().ForEach(s =>
+                        speciesInPatches[patchEntry.Key].Add(s.Key, s.Value));
+                }
+            }
+
+            return speciesInPatches;
+        }
+
+        /// <summary>
+        ///   Computes the final population of a species and its splitted forms, by patch.
+        /// </summary>
+        /// <remarks>
+        ///   <para>
+        ///     Species are only returned if their population is above 0.
+        ///   </para>
+        /// </remarks>
+        public Dictionary<Patch, Dictionary<Species, long>> GetPopulationsByPatch(Species species,
+            bool resolveMigrations = false, bool resolveSplits = false, bool includeNewSpecies = true)
```

Comment:
I find this naming pretty confusing so this is given the species but it also returns split off (ie. different species objects) from this? I'd rename this method. And add xml docs for all the parameters to explain this better.

Reply:
Would `GetOffspringPopulationByPatch` look nice to you? During a splitoff, even the main branch *technically* becomes an offspring, I'd say.

Reply:
Maybe "clade" or some other species family tree related term would be better than offspring? I've never seen species being referred to as offspring.

Reply:
Yeah, I guess "clade" is probably more accurate a term. I'll use that for the time being, and I'll try to catch a biologist tomorrow to confirm.